test/functional/page_objects/dashboard_page_controls.ts

```diff
@@ -234,6 +235,30 @@ export class DashboardPageControls extends FtrService {
     return controlElement;
   }
 
+  public async createControl({
+    controlType,
```

Comment:
Introduced this generic `createControl` function to avoid the duplicated code we had previously with individual `create<OptionsList/RangeSlider/TimeSlider>Control` functions - this is why so many seemingly-unrelated test files were edited 😆